bin/modules-docs.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var https = require('https');
+var logger = require('console');
+
+var docs = path.join(__dirname, '..', 'docs', 'modules');
+var moduleReadmes = {
+    'geval': ['Raynos/geval', '2.1.1', 'README.md'],
```

Comment:
hard coding the version number is going to be frustrating. I recommend you read it from the README and ideally query npm for the latest version matching the semver range.

Alternatively just fetch the HEAD from the github repository.